{"uid":"eba6d3eeef44a7ac","name":"Validate successful update of chatbot API details -- @1.1 ","fullName":"Update Chatbot API Details using Genr3d API: Validate successful update of chatbot API details","historyId":"506910af4d3e294515e00e35b629cc36","time":{"start":1757682287139,"stop":1757685715190,"duration":3428051},"status":"failed","statusMessage":"AssertionError: Chatbot status did not become AVAILABLE after 250 attempts\n","statusTrace":"  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Update_API_Information_of_an_existing_chatbot.feature.py\", line 85, in step_impl\n    assert False, f\"Chatbot status did not become AVAILABLE after {max_retries} attempts\"\n           ^^^^^\n","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"AssertionError: Chatbot status did not become AVAILABLE after 250 attempts\n","statusTrace":"  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Update_API_Information_of_an_existing_chatbot.feature.py\", line 85, in step_impl\n    assert False, f\"Chatbot status did not become AVAILABLE after {max_retries} attempts\"\n           ^^^^^\n","steps":[{"name":"Given User sends PUT request to update chatbot API details for chatbot ID with environment ID and endpoint \"http://34.36.66.146/stream_messages\"","time":{"start":1757682287140,"stop":1757685715178,"duration":3428038},"status":"failed","statusMessage":"AssertionError: Chatbot status did not become AVAILABLE after 250 attempts\n","statusTrace":"  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\model.py\", line 1991, in run\n    match.run(runner.context)\n  File \"C:\\Users\\Noman\\PycharmProjects\\generativesecurity\\.venv\\Lib\\site-packages\\behave\\matchers.py\", line 105, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\Update_API_Information_of_an_existing_chatbot.feature.py\", line 85, in step_impl\n    assert False, f\"Chatbot status did not become AVAILABLE after {max_retries} attempts\"\n           ^^^^^\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false},{"name":"When the user receives the chatbot API update response","time":{"start":1757685715189,"stop":1757685715189,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false},{"name":"Then the status code returned should be 200 for successful update","time":{"start":1757685715190,"stop":1757685715190,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false},{"name":"And the response should contain a valid chatbotID","time":{"start":1757685715190,"stop":1757685715190,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false},{"name":"And the response should contain a \"status\" equal to \"API_updated\"","time":{"start":1757685715190,"stop":1757685715190,"duration":0},"status":"skipped","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":0,"attachmentStep":false}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":false,"attachmentsCount":0,"stepsCount":5,"attachmentStep":false},"afterStages":[],"labels":[{"name":"severity","value":"normal"},{"name":"tag","value":"Test"},{"name":"feature","value":"Update Chatbot API Details using Genr3d API"},{"name":"framework","value":"behave"},{"name":"language","value":"cpython3"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"chatbot_url","value":"http://34.36.66.146/stream_messages"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"6508bfe4b4555892","status":"broken","statusDetails":"requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: https://identitytoolkit.googleapis.com/v1/accounts:signInWithPassword?key=AIzaSyD08yxn7V_YAfkAY8mKDTYjfKV_qoPrdRQ\n","time":{"start":1757682177158,"stop":1757682177815,"duration":657}}],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":["Test"]},"source":"eba6d3eeef44a7ac.json","parameterValues":["http://34.36.66.146/stream_messages"]}